.github/workflows/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Load Actions Test
+on:
+  push:
+    branches:
+      - DEVOPS-79
+jobs:
+  action-whitelist:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
Just for consistency, I still prefer the hash even on GitHub's native actions. Or maybe that's the point, so that this test verifies itself and throws an error?

Reply:
updated to the latest commit hash, and added that to the approved.json